source/imaer-gml/src/main/java/nl/overheid/aerius/gml/v5_0/togml/GMLVersionWriterV50.java

```diff
@@ -120,7 +121,7 @@ private SituationMetadata getSituation(final MetaDataInput input) throws AeriusE
       situation = new SituationMetadata();
       situation.setName(input.getName());
       situation.setReference(input.getReference());
-      situation.setSituationType(input.getSituationType());
+      situation.setSituationType(input.getSituationType() != null ? LegacySituationType.safeValueOf(input.getSituationType().name()) : null);
```

Comment:
Hah, this is a good one. In this case I guess it should write `NETTING` instead of `OFF_SITE_REDUCTION` to still fit the 5.0 XSD?